integration/src/integration/java/pl/allegro/tech/hermes/frontend/HeadersPropagatorConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package pl.allegro.tech.hermes.frontend;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import pl.allegro.tech.hermes.frontend.publishing.metadata.HeadersPropagator;
+import pl.allegro.tech.hermes.integration.metadata.TraceHeadersPropagator;
+
+@Configuration
+public class HeadersPropagatorConfiguration {
+
+    @Bean
+    @Primary
```

Comment:
I have to leave it as it is, because otherwise tests are failing with for example `                org.springframework.beans.factory.NoUniqueBeanDefinitionException: No qualifying bean of type 'pl.allegro.tech.hermes.common.kafka.KafkaNamesMapper' available: expected single matching bean but found 2: testKafkaNamesMapper,prodKafkaNamesMapper
`